backend/stakepoold/rpc/api.proto

```diff
@@ -40,6 +41,15 @@ message GetLiveTicketsResponse {
 	repeated Ticket tickets = 1;
 }
 
+message GetTicketInfoRequest {
+    string Hash = 1;
+}
+message GetTicketInfoResponse {
+	string MultiSigAddress = 1;
+	string VspFeeAddress = 2;
+	string OwnerFeeAddress = 3;
+}
```

Comment:
We might want to start tracking txids in the db, as we want to become more self-sufficient.  Although I am aware that you want to reduce the communication between the db and stakepoold, because it is on another machine.  A centralized area to track tickets is needed, imo.